src/main/java/net/minecraftforge/client/event/RenderTooltipEvent.java

```diff
@@ -48,14 +49,16 @@
     @Nonnull
     protected final ItemStack stack;
     protected final List<? extends ITextProperties> lines;
+    protected final MatrixStack matrixStack;
```

Comment:
Correct me if I am wrong, but I don't believe you added a getter to actually *get* the matrix stack?